src/transformers/trainer.py

```diff
@@ -995,8 +995,8 @@ def _wrap_model(self, model, training=True):
                 find_unused_parameters = True
             model = nn.parallel.DistributedDataParallel(
                 model,
-                device_ids=[self.args.local_rank],
-                output_device=self.args.local_rank,
+                device_ids=[self.args.local_rank] if self.args.n_gpu != 0 else None,
+                output_device=self.args.local_rank if self.args.n_gpu != 0 else None,
```

Comment:
```suggestion
                device_ids=[self.args.local_rank] if self.args._n_gpu != 0 else None,
                output_device=self.args.local_rank if self.args._n_gpu != 0 else None,
```
It needs to be the private argument, not the public one.